js/index.js

```diff
@@ -1,7 +1,91 @@
-// Iteration 1: Names and Input
+console.log("hello");
 
+// Iteration 1 Names and Input 
 
-// Iteration 2: Conditionals
+let hacker1 = "Benjamin";
 
+console.log(`drivers name is ${hacker1}`);
 
-// Iteration 3: Loops
\ No newline at end of file
+let hacker2 = "Maria";
+
+console.log(`navigators name is ${hacker2}`);
+
+// Iteration 2 Conditionals 
+
+if (hacker1.length > hacker2.length) {
+    console.log(`the driver hast the longest name, it has ${hacker1.length} characters`);
+} else if (hacker1.length < hacker2.length) {
+    console.log(`the navigator hast the longest name, it has ${hacker2.length} characters`);
+} else if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names ${hacker2.length} characters!`);
+}
+
+// Iteration 3: Loops
+
+let capitalizedHacker1 = hacker1.toUpperCase();
+
+console.log(capitalizedHacker1);
+
+/*
+let hacker1Splitted = capitalizedHacker1.split(' ') ;
+
+console.log(hacker1Splitted) ;
+*/
+
+let result = '';
+
+for (i = 0; i < hacker1.length; i++) {
+    result += capitalizedHacker1[i] + ' ';
+}
+
+console.log(result);
+
+let result2 = '';
+
+for (i = hacker2.length - 1; i > -1; i--) {
+    result2 += hacker2[i];
+}
+
+console.log(result2);
+
+var arrNames = [hacker1, hacker2];
+
+arrNames.sort();
+
```

Comment:
nice approach! You could also have used localeCompare() to sort the words on lexicographic order in case you are interested :)